static_report/src/components/ComparisonReport/CRTableList/CRTableListRowsSummary.tsx

```diff
@@ -0,0 +1,39 @@
+import { Flex, Text, Icon } from '@chakra-ui/react';
+import {
+  FiArrowUpCircle,
+  FiArrowDownCircle,
+  FiArrowRight,
+} from 'react-icons/fi';
+
+export function CRTableListRowsSummary({
+  base,
```

Comment:
same here. 
More descriptive props for base/target
it's hard to know purely from its typing alone

Reply:
done